### PR TITLE
[Chore] Bump morley

### PR DESCRIPTION
**Pull request by lierdakil**
Draft because pulls dependencies from branches.

## Description

Problem: new morley &c are ready

Solution: update

Problem: CI is setup to test on Lima still

Solution: update to test on Mumbai instead.

## Related issue(s)

Part of https://issues.serokell.io/issue/TM-674

#### Related changes (conditional)

- Tests (see [short guidelines](../blob/master/CONTRIBUTING.md#tests))
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - I checked whether I should update the docs and did so if necessary:
    - [ ] [README](../blob/master/README.md)
    - [ ] Haddock
    - [ ] [docs/](../blob/master/docs/)
  - [ ] I updated [changelog](../blob/master/ChangeLog.md) unless I am sure my changes are
        really minor.

#### Stylistic guide (mandatory)

- [x] My commits comply with [the following policy](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [x] My code complies with the [style guide](../blob/master/docs/code-style.md).

### CODE REVIEW

File: bats/tzbtc.bats [hunk 1]
```diff
@@ -37,7 +37,7 @@ setup () {
 
 @test "invoking tzbtc 'printInitialStorage' command" {
   result="$($tzbtc printInitialStorage --owner-address tz1f1S7V2hZJ3mhj47djb5j1saek8c2yB2Cx)"
-  [ "$result" == '{ { Elt 0x0501000000056f776e6572 0x050a000000160000d476acd953eb55d38c398c85c3f53e19b62b167a }; { CDR; NIL operation; PAIR }; 0; False }' ]
+  [ "$result" == 'Pair { Elt 0x0501000000056f776e6572 0x050a000000160000d476acd953eb55d38c398c85c3f53e19b62b167a } (Pair { CDR; NIL operation; PAIR } (Pair 0 False))' ]
```
Comment (pasqu4le) [inline bats/tzbtc.bats:40]: Uh-oh, do you know why this was needed?

AFAIK either syntax used here should work :thinking:

Comment (lierdakil) [reply bats/tzbtc.bats:40]: We don't linearize untyped right combs any more, as in general that's not quite correct, and typed value is printed unoptimized by default. I could print an optimized value instead, although it's a bit involved with Lorentz values (as is the case here)

Comment (lierdakil) [reply bats/tzbtc.bats]: I've pushed 7ae9e983e058b5a25faad3ea3936bd36b73389ac amending this.

Comment (pasqu4le) [reply bats/tzbtc.bats]: got it, thanks for the explanation :ok_hand: 

Let's keep the version with the optimization then, if you don't mind

**Review by pasqu4le — APPROVED** [review]

**Pull request merged**